velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMethods.java

```diff
@@ -0,0 +1,368 @@
+package me.leoko.advancedban.velocity;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonNull;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.proxy.Player;
+import de.leonhard.storage.LightningBuilder;
+import de.leonhard.storage.Yaml;
+import de.leonhard.storage.internal.settings.ConfigSettings;
+import jdk.nashorn.internal.parser.JSONParser;
+import me.leoko.advancedban.MethodInterface;
+import me.leoko.advancedban.Universal;
+import me.leoko.advancedban.manager.PunishmentManager;
+import me.leoko.advancedban.manager.UUIDManager;
+import me.leoko.advancedban.utils.Punishment;
+import me.leoko.advancedban.utils.tabcompletion.TabCompleter;
+import me.leoko.advancedban.velocity.event.PunishmentEvent;
+import me.leoko.advancedban.velocity.event.RevokePunishmentEvent;
+import me.leoko.advancedban.velocity.listener.CommandReceiverVelocity;
+import net.kyori.text.TextComponent;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public class VelocityMethods implements MethodInterface {
+
+    private final File configFile = new File(getDataFolder(), "config.yml");
+    private final File messageFile = new File(getDataFolder(), "Messages.yml");
+    private final File layoutFile = new File(getDataFolder(), "Layouts.yml");
+    private final File mysqlFile = new File(getDataFolder(), "MySQL.yml");
+    private Yaml config;
+    private Yaml messages;
+    private Yaml layouts;
+    private Yaml mysql;
+
+    @Override
+    public void loadFiles() {
+        config = LightningBuilder.fromFile(configFile).addInputStreamFromResource(configFile.getName()).setConfigSettings(ConfigSettings.PRESERVE_COMMENTS).createYaml();
+        messages = LightningBuilder.fromFile(messageFile).addInputStreamFromResource(messageFile.getName()).setConfigSettings(ConfigSettings.PRESERVE_COMMENTS).createYaml();
+        layouts = LightningBuilder.fromFile(layoutFile).addInputStreamFromResource(layoutFile.getName()).setConfigSettings(ConfigSettings.PRESERVE_COMMENTS).createYaml();
+        if (mysqlFile.exists()) {
+            mysql = LightningBuilder.fromFile(mysqlFile).setConfigSettings(ConfigSettings.PRESERVE_COMMENTS).createYaml();
+        } else mysql = config;
+    }
+
+    @Override
+    public String getFromUrlJson(String url, String key) {
+        try {
```

Comment:
Although I know this is somewhat outside the scope of this PR, this method's body is much duplicated. Rather than write it a 3rd time, perhaps there is a better way to consolidate all of it?